docs/source/getting_started/installation.md

```diff
@@ -0,0 +1,61 @@
+## Installation
+Elyra can be installed via PyPi:
+
+### Prerequisites
+* [NodeJS 12+](https://nodejs.org/en/)
+* [Python 3.X](https://www.anaconda.com/distribution/)
+
+##### Optional
+* [Anaconda](https://www.anaconda.com/distribution/) 
+* [Docker](https://docs.docker.com/install//) - If using the docker image
+
+#### JupyterLab support
+
+* [JupyterLab](https://github.com/jupyterlab/jupyterlab) 1.x is supported on **Elyra 0.10.0 and below**
+* [JupyterLab](https://github.com/jupyterlab/jupyterlab) 2.x is supported on **Elyra 0.11.0-rc0 and above**
+
+via PyPi:
+```bash
+pip install elyra && jupyter lab build
+```
+Note: Ubuntu and CentOS users may need to use `pip3 install elyra` 
+
+### Verify Installation 
+```bash
+jupyter serverextension list
+```
+Should output:
+```
+config dir: /usr/local/etc/jupyter
+    elyra  enabled
+    - Validating...
+      elyra  OK
```

Comment:
Let's go ahead and add the latest elyra build (0.10.1) - now that that's included in this output (thank you!).

Reply:
Fixed.